html/navbar.html

```diff
@@ -1,31 +1,35 @@
 {{define "navbar"}}
+<script type="text/javascript" src="/static/js/signal.js"></script>
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-    <a class="navbar-brand" href="/">{{DeviceName}}</a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-    </button>
+  <a class="navbar-brand" href="/">{{DeviceName}}</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+  </button>
 
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <ul class="navbar-nav mr-auto">
-          <li class="nav-item">
-            <a class="nav-link" href="/camera">Camera</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/disk-memory">Disk &amp; Memory</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/network">Network</a>
-          </li>
-             <li class="nav-item">
-            <a class="nav-link" href="/wifi-networks">WiFi Networks</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/speaker">Speaker Test</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/location">Location</a>
-          </li>
-        </ul>
-    </div>
-    </nav>
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="/camera">Camera</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/disk-memory">Disk &amp; Memory</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/network">Network</a>
+      </li>
+         <li class="nav-item">
+        <a class="nav-link" href="/wifi-networks">WiFi Networks</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/speaker">Speaker Test</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/location">Location</a>
+      </li>
+    </ul>
+    <span class="navbar-text">
+      Signal: <span id="signal-strength"> - </span> dB
```

Comment:
Can we use an icon here instead of the word "Signal:"?

Reply:
extra bonus points for using an icon which changes according to the signal level. e.g. https://fontawesome.com/icons?d=gallery&q=signal

Reply:
Added svg